src/components/medication-dispense-review.scss

```diff
@@ -2,6 +2,13 @@
 @use '@carbon/styles/scss/type';
 @import '~@openmrs/esm-styleguide/src/vars';
 
+.medicationDispenseReviewContainer {
+  :global(.cds--css-grid) {
+    padding-left:0 !important;
+    padding-right:0 !important;
+  }
```

Comment:
I did this to reduce the grid marginings/padding, which was quite large, not sure if is the "proper" way to do this... also, @pirupius  I wasn't sure where the rest of the CSS below in this file was used?  I tried removing it and didn't see any obvious difference?